www/script/templates/popups.hbs

```diff
@@ -26,16 +26,7 @@
 {{#showPendingJobs}}
     <h3>Pending Build Requests</h3>
     <ul class="pending-job-list">
-        <li>
```

Comment:
@opdude I dont think we want to put it back

this will allows people to cancel all the builds for that builder